maint-lib/stagelib/envglobals.py

```diff
@@ -105,3 +106,19 @@ def exportGoArchEnvVar():
         os.environ['GO_ARCH'] = _CEPH_ARCH_TO_GOLANG_ARCH_CONVERSIONS[arch]
     else:
         os.environ['GO_ARCH'] = arch
+
+
+def exportCephVersionSpecifics():
+    """ We need to extract a possible CEPH_POINT_RELEASE from CEPH_VERSION
+    It could be under the two following syntax :
+     - package_name-package_version
+     - package_name=package_version"
+    """
+    os.environ["CEPH_POINT_RELEASE"] = ""
+    original_ceph_version = getEnvVar("CEPH_VERSION")
+    if original_ceph_version:
+        for separator in ["-", "="]:
+            if separator in original_ceph_version:
+                splitted = original_ceph_version.split(separator)
+                os.environ["CEPH_VERSION"] = splitted[0]
```

Comment:
The code should not be reassigning user-specified environment variables. That would be very confusing to users to have a variable they personally set change values during runtime. We need to extract this into 2 parts.